openfermioncirq/gates/four_qubit_gates.py

```diff
@@ -178,8 +178,8 @@ def _circuit_diagram_info_(self, args: cirq.CircuitDiagramInfoArgs
         if args.use_unicode_characters:
             wire_symbols = ('⇅', '⇅', '⇵', '⇵')
         else:
-            wire_symbols = ('/\\ \/',
-                            '/\\ \/',
+            wire_symbols = (r'/\ \/',
+                            r'/\ \/',
                             '\/ /\\',
                             '\/ /\\')
```

Comment:
Can you  make the same change for the latter two strings too?

Reply:
Nope. Raw strings can't end in a single backslash.